docs/how-to/default-profile.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Default profile
+legacyUrl: https://docs.ipfs.io/guides/examples/default-profile/
+description: Hands-on guide for rolling back to the default profile
+---
+
+# Configure a default profile and switching between profiles
+
+---
```

Comment:
```suggestion
```

HRs aren't necessary. Adding extra styling like this makes it difficult to be consistent across all pages in this site.